torch/nn/modules/conv.py

```diff
@@ -9,10 +9,36 @@
 from .. import init
 from .module import Module
 from .utils import _single, _pair, _triple, _reverse_repeat_tuple
+from torch._torch_docs import reproducibility_notes
 
 from ..common_types import _size_1_t, _size_2_t, _size_3_t
 from typing import Optional, List, Tuple
 
+convolution_notes = \
+    {"groups_note": """* :attr:`groups` controls the connections between inputs and outputs.
+      :attr:`in_channels` and :attr:`out_channels` must both be divisible by
+      :attr:`groups`. For example,
+
+        * At groups=1, all inputs are convolved to all outputs.
+        * At groups=2, the operation becomes equivalent to having two conv
+          layers side by side, each seeing half the input channels,
+          and producing half the output channels, and both subsequently
+          concatenated.
+        * At groups= :attr:`in_channels`, each input channel is convolved with
+          its own set of filters (of size
+          : math: `\left\lfloor\\frac{out\_channels}{in\_channels}\\right\\rfloor`).""",  # noqa: W605
```

Comment:
:math:

Reply:
The expression could add \\text functions:

```
\left\lfloor\frac{\text{out\_channels}}{\text{in\_channels}}\right\rfloor
```

are the double backslashes just a docs parsing thing?